src/poliastro/twobody/orbit.py

```diff
@@ -1094,9 +1095,14 @@ def plot(self, label=None, use_3d=False):
             Label for the orbit, defaults to empty.
         use_3d : bool, optional
             Produce a 3D plot, default to False.
-
+        static : bool, optional
+            Produce a static image of the figure, default to false
         """
-        if use_3d:
+        from poliastro.plotting.static import StaticOrbitPlotter
+
+        if static:
+            return StaticOrbitPlotter().plot(self, label=label)
+        elif use_3d:
```

Comment:
```suggestion
        if use_3d:
```

Reply:
Why? Also, perhaps we should raise an error if `static` and `use_3d` are given.